client/src/main/scala/drt/client/components/PortConfigPage.scala

```diff
@@ -0,0 +1,164 @@
+package drt.client.components
+
+import drt.client.modules.GoogleEventTracker
+import drt.client.services.SPACircuit
+import drt.shared.CrunchApi.MillisSinceEpoch
+import drt.shared.FlightsApi.{QueueName, TerminalName}
+import drt.shared.PassengerSplits.QueueType
+import drt.shared._
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.{Callback, ScalaComponent}
+
+object PortConfigPage {
+
+  case class Props()
+
+  val component = ScalaComponent.builder[Props]("ConfigPage")
+    .render_P(_ =>
+      <.div(^.className := "port-config", <.h3("Port Config"), PortConfigDetails())
+    )
+    .componentDidMount(p => Callback {
+      GoogleEventTracker.sendPageView(s"port-config")
+    })
+    .build
+
+  def apply(): VdomElement = component(Props())
+}
+
+object PortConfigDetails {
+
+  case class Props()
+
+  val component = ScalaComponent.builder[Props]("ConfigDetails")
+    .render_P(p => {
+      val airportConfigRCP = SPACircuit.connect(_.airportConfig)
+      <.div(
+        airportConfigRCP(airportConfigMP =>
+          <.div(airportConfigMP().renderReady(config => <.div(
+            config.terminalNames.map(tn =>
+              <.div(
+                <.h2(tn),
+                <.div(^.className := "container config-container",
+                  <.h4("Min / Max Desks by hour of day"),
```

Comment:
Might be worth adding about egates eg "Min / Max desks / egate banks by hour of day" here just for clarity